client/src/app/modules/events/event-filter/event-filter.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="row">
+    <div class="col-12 col-md-8">
+        <input type="text" [(ngModel)]="searchTerm" placeholder="Search in events..." (keydown)="searchEventsOnEnter($event)"/>
+    </div>
+    <div class="col-12 col-md-4">
+        <button class="pull-right" placement="top" ngbTooltip="Create" (click)="createEvent()">
+            <i class="fa fa-plus"></i>
+        </button>
+        <button class="pull-right" placement="top" ngbTooltip="Joined" [ngClass]="joined ? 'active': ''" (click)="toggleJoined()">
```

Comment:
I believe the ngClass can be written nicer like this [ngClass]="{ 'active' : joined }" 
See [docs](https://angular.io/api/common/NgClass#description)

Reply:
fixed